colibre/scripts/density_temperature_species.py

```diff
@@ -0,0 +1,191 @@
+"""
+Makes a rho-T plot. Uses the swiftsimio library.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from swiftsimio import load
+
+from unyt import mh, cm
+from matplotlib.colors import LogNorm, ListedColormap, BoundaryNorm
+from matplotlib.animation import FuncAnimation
+from matplotlib.cm import get_cmap
+
+# Set the limits of the figure.
+density_bounds = [10 ** (-9.5), 1e6]  # in nh/cm^3
+temperature_bounds = [10 ** (0), 10 ** (9.5)]  # in K
+dustfracs_bounds = [-2, 0]  # In metal mass fraction
```

Comment:
Is it possible to specify these in linear space (`dustfracs_bounds = [1e-2, 1e0]`), and use that throughout with  logarithmic normalizations instead of linear ones on log quantities? That's what we do in all other cases so it would be nice here too.